WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -404,6 +404,14 @@ private extension ProductsViewController {
     func registerTableViewCells() {
         tableView.register(ProductsTabProductTableViewCell.self)
     }
+
+    func showOrHideToolBar(resultController: ResultsController<StorageProduct>) {
+        if resultsController.isEmpty {
+            toolbar.isHidden = true
+        } else {
+            toolbar.isHidden = false
+        }
```

Comment:
I think this could be 
```suggestion
        toolbar.isHidden = resultsController.isEmpty
```

Reply:
Good catch! Added!